Gem/Code/Source/Weapons/SceneQuery.cpp

```diff
@@ -79,8 +80,9 @@ namespace MultiplayerSample
             auto ignoreEntitiesFilterCallback =
                 [&filter, networkEntityManager](const AzPhysics::SimulatedBody* body, [[maybe_unused]] const Physics::Shape* shape)
             {
-                // Exclude the bodies from another rewind frame
-                if ((filter.m_rewindFrameId != Multiplayer::InvalidHostFrameId)
+                // Exclude the dynamic bodies from another rewind frame
+                if (filter.m_rewindFrameId != Multiplayer::InvalidHostFrameId
+                    && body->GetNativeType() != PhysX::NativeTypeIdentifiers::RigidBodyStatic
```

Comment:
It may be better to use `&&  (body->GetFrameId() != AzPhysics::SimulatedBody::UndefinedFrameId)` instead of the NativeType check in case of there are non-static rigid bodies that are intentionally not rewound